### PR TITLE
Review user like me

### PR DESCRIPTION
**Pull request by anhntv-0911**
==> Tittle:Review user like me

==> Redmine:https://edu-redmine.sun-asterisk.vn/issues/17408

==> Attachment File:
Screenshot from 2019-09-23 11-25-52
Screenshot from 2019-09-23 11-26-13

### CODE REVIEW

**Comment by anhntv-0911** [comment]
ready

**Comment by anhntv-0911** [comment]
ready

File: app/assets/stylesheets/review_liked_me.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the ReviewLikedMe controller here.
```
Comment (hungct-1121) [inline app/assets/stylesheets/review_liked_me.scss]: bỏ file này đi em

File: app/controllers/review_liked_me_controller.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    redirect_to review_liked_me_index_path
+  end
+
+  def index
+    @users = User.info_user_like_me(current_user).page(params[:page]).per Settings.page
+  end
+
+  def react
+    reaction = current_user.send(@react.to_sym, @pass_user)
+    if reaction && reaction.like?
```
Comment (hungct-1121) [inline app/controllers/review_liked_me_controller.rb]: thêm dòng trống phía trên

**Comment by anhntv-0911** [comment]
ready

**Pull request merged**